src/include/migraphx/op/argmax.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_ARGMAX_HPP
+#define MIGRAPHX_GUARD_OPERATORS_ARGMAX_HPP
+
+#include <migraphx/operation.hpp>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/par_dfor.hpp>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct argmax
+{
+    int axis = 0;
```

Comment:
This should probably be `int64_t`.

Reply:
That is the type indicated by onnx operator and this can avoid the unnecessary conversion from int to int64 when parsing onnx file.

Reply:
changed to int64_t